components/event-block/EventBlockChapters.vue

```diff
@@ -0,0 +1,235 @@
+<template>
+  <div class="event-block-chapters">
+    <p class="event-block-chapters__title">
+      {{ title }}
+    </p>
+
+    <swiper :options="swiperOptions">
+      <swiper-slide
+        v-for="item in items"
+        :key="item.slug"
+      >
+        <nuxt-link
+          :to="`${currentRoute}/${item.slug}`"
+          class="event-block-chapters__link"
+        >
+          <span class="event-block-chapters__item-title">{{ item.title }}</span>
+          <img
+            v-if="item.cover"
+            :data-src="`${item.cover.url}?auto=compress&w=470`"
+            alt=""
+            class="event-block-chapters__item-image swiper-lazy"
+          >
+          <div
+            v-else
+            class="event-block-chapters__item-placeholder"
+          />
+          <div class="swiper-lazy-preloader swiper-lazy-preloader-white" />
+        </nuxt-link>
+      </swiper-slide>
+
+      <div
+        slot="button-prev"
+        class="swiper-button-prev"
+      />
+
+      <div
+        slot="button-next"
+        class="swiper-button-next"
+      />
+
+      <div
+        slot="scrollbar"
+        class="swiper-scrollbar"
+      />
+    </swiper>
+  </div>
+</template>
+
+<script>
+import { Swiper, SwiperSlide, directive } from 'vue-awesome-swiper';
+import 'swiper/css/swiper.css';
+
+export default {
+  components: {
+    Swiper,
+    SwiperSlide,
+  },
+  props: {
+    title: String,
+    items: Array,
+  },
+  data() {
+    return {
+      swiperOptions: {
+        slidesPerView: 3,
+        preloadImages: false,
+        lazy: {
+          loadOnTransitionStart: true,
+        },
+        watchSlidesProgress: true,
+        watchSlidesVisibility: true,
+        mousewheel: {
+          forceToAxis: true,
+        },
+        keyboard: {
+          enabled: true,
+          onlyInViewport: true,
+        },
+        navigation: {
+          nextEl: '.swiper-button-next',
+          prevEl: '.swiper-button-prev',
+        },
+        scrollbar: {
+          el: '.swiper-scrollbar',
+        },
+        breakpoints: {
+          600: {
+            slidesPerView: 2.8,
+            spaceBetween: 20,
+          },
+          0: {
+            slidesPerView: 1.2,
+            spaceBetween: 10,
+          },
+        },
+      },
+    };
+  },
+  computed: {
+    currentRoute () {
+      return this.$route.path;
+    },
+  },
+};
+</script>
+
+<style>
```

Comment:
Could you indent the whole `<style>` content? Makes it consistent with the `<template>` and foldable/collapsible for more code editors.